Alcancia-io/alcancia
# Feature/metamap implementation

## Summary
Add metamap implementation SDK, added id to queries and changes userId for authId. 
The status needs to be implemented still.

TODO: @manu-yaff 
  Implement Cancel KYC enpoint when the user cancels 
  Show in the dashboard the status of the verification

## Requirements
Test it on the branch Feature/OINK-844 of the backend
make sure you reaplace your `.env` with the latest version

## Type of change
- [ x ] feature

## How to test?
Run the app, login and try to make a deposit through the transfer button it should take you to the KYC and after you finalize it should show you a message of success.

## How has this been tested?


- [ ] Check if the kycStatus in the table changes

## Screenshots
_add screenshots if need it_

## Checklist (for the reviewer)
